srtcore/buffer.h

```diff
@@ -441,17 +485,42 @@ class CRcvBuffer
 private:
    bool scanMsg(ref_t<int> start, ref_t<int> end, ref_t<bool> passack);
 
+   int shift(int basepos, int shift) const
+   {
+       return (basepos + shift) % m_iSize;
+   }
+
+   // Simplified versions with ++ and --; avoid using division instruction
+   int shift_forward(int basepos) const
+   {
+       if (++basepos == m_iSize)
+           return 0;
+       return basepos;
+   }
+
+   int shift_backward(int basepos) const
```

Comment:
Function names do not follow the convention of CRcvBuffer.
Consider using something like `incPos()` and `decPos`, or `shiftForward` and `shiftBackward()`

Reply:
Actually you are right, didn't think about this at that time.